test/text/vtt_text_parser_unit.js

```diff
@@ -171,28 +171,28 @@ describe('VttTextParser', () => {
   });
 
   it('rejects invalid time values', () => {
-    errorHelper(shaka.util.Error.Code.INVALID_TEXT_CUE,
+    verifyHelper([],
```

Comment:
I think this is way too permissive.  This basically allows anything and just ignores things we can't figure out.  It is preferable to throw errors for invalid content so content authors know their content is broken and don't get confused about why their cues aren't appearing.  I think you should restrict this to only ignoring extra newlines.

Reply:
Ignoring only extra newlines is difficult because of the way VTT files are structured. The `\n\n` sequence is sacred. The parser uses `\n{2,}` to split the file into blocks, which appears to be the right way to do it. When this is fed cues with extra newlines you get things like [this (jsfiddle)](https://jsfiddle.net/bloomtom/ocre4qud/2/). I don't know we can ignore _only_ extra newlines without ignoring all bad cue data while not also breaking significantly from how the VTT spec defines how a parser should be built.

I do agree that something should be logged. How about:
```js
shaka.log.warning(
    'Failed to parse VTT time code. Cue skipped:', id, text);
```

Reply:
I've tentatively implemented the above with 9755442.